pyface/ui/wx/grid/tests/test_simple_grid_model.py

```diff
@@ -56,9 +56,6 @@ def test_get_value(self):
 
     def test_is_cell_empty(self):
 
-        rows = self.model.get_row_count()
-        columns = self.model.get_column_count()
-
         self.assertEqual(
```

Comment:
the second assert below is a hard coded magic number 10, perhaps we should just use rows+1/columns+1 there?

Reply:
fixed. waiting to see if CI is happy.